### ISSUE TITLE
underwater multi-camera systems

### ISSUE THREAD

**Issue by 342600694** [ohkaihun/Calibraion_ctool#1]
Hello, does this toolbox support calibration for underwater multi-camera systems?

**Comment by ohkaihun** [comment]
Yes, this calibration program is specifically designed for underwater multi-camera system. It supports various types of cameras and allows you to choose the number of cameras as well. I will also provide you with some guidance to help you use this tool more effectively.

**Comment by 342600694** [comment]
> Yes, this calibration program is specifically designed for underwater multi-camera system. It supports various types of cameras and allows you to choose the number of cameras as well. I will also provide you with some guidance to help you use this tool more effectively.

Hello, I have a question. How should I use a refraction-corrected camera for pose estimation tasks, since the camera is no longer based on the pinhole model?

**Comment by ohkaihun** [comment]
Hello, according to our pipeline, the calibration process is divided into two steps. In the first step, we first perform the calibration for the intrinsic parameters and poses of the multi-camera system which are all pre-calibrated in air. And this step does not consider the effects of refraction（air/glass or glass/water) because we doesn't install waterproof ports. In the second step, waterproof ports were installed on all cameras(ports can be dome/flat). Then, we perform the calibration for the offset between the camera center and waterproof ports which are calibrated only in water.  Our calibration results can be used for subsequent ray-based studies.
